index.js

```diff
@@ -0,0 +1,79 @@
+var express = require("express");
+// FS (file ssystem) provided by node by default
+// FS used to read and write files (data.json)
+var fs = require("fs");
+var bodyParser = require("body-parser");
+var app = express();
+
+// set up ejs engine
+app.set("view engine", "ejs");
+// set up body-parser
+app.use(bodyParser.urlencoded({ extended:false }));
+
+// link to folder with "static pages to use like css/images/html"
+app.use(express.static("src"));
+
+// Below renders page
+app.get("/", function(req, res) {
+  res.render("index.ejs");
+});
+
+app.get("/site/about", function(req, res) {
```

Comment:
typically you would just use /about /contact for pages that are global for the site. However you maintained the context by having their addresses be /site/ which is the most important thing.